Update about-snowflake-native-app.md (#6052)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud-integrations/about-snowflake-native-app.md

App users are able to access all information that's available to the API service token.

## Procurement

> The dbt Snowflake Native App is available on the [Snowflake Marketplace](https://app.snowflake.com/marketplace/listing/GZTYZSRT2R3). With purchase, users will have access to the Native App and a dbt Cloud account that's on the Enterprise plan. It is also available to all dbt Cloud Enterprise customers. If you are interested, speak to your Enterprise Account Manager. 

Reviewer: ```suggestion
The dbt Snowflake Native App is available on the [Snowflake Marketplace](https://app.snowflake.com/marketplace/listing/GZTYZSRT2R3). Purchasing it includes access to the Native App and a dbt Cloud account that's on the Enterprise plan. Existing dbt Cloud Enterprise customers can also access it. If interested, contact your Enterprise account manager.
```
